### DISCUSSION TITLE
Frigate annotations on RTMP re-stream?

### DISCUSSION THREAD

**Discussion by diegobaccino** [blakeblackshear/frigate#1573] [Ask A Question]
I need to display the real-time annotations (especially Boundary Box) in the RTMP feed for downstream consumers like it's done in the UI. Is this possible? Is there an attribute I can use in the ffmpeg output_args or something? I appreciate your help :)

**Comment by blakeblackshear** [comment]
"need" is a strong word. I don't know of a good way to do this.

**Reply by diegobaccino** [reply]
Gotcha. I think this would be a neat feature. Thanks for the quick reply.
